infra/azure/README.md

```diff
@@ -106,6 +122,9 @@ Deploy unmanaged resources by running
 ./bootstrap.sh deploy_unmanaged
 ```
 
+If the final letsencrypt step in deploy_unmangaed fails, copy the kubectl secret from
+stdout and apply it manually.
```

Comment:
Could you elaborate on what error you saw here and what secret needs to be copied?

Reply:
The elaboration is quite verbose so if I need to simplify again let me know. Basically the letsencrypt.sh only can be run a certain number of times before it permanently fails within a certain window. Without the intervention specified above we were unable to retrieve the secret and have it applied properly to the kubernetes cluster.

So we manually retrieved it through stdout and applied it reverting the changes. If the deploy_unmanaged step runs successfully first time then this is not required. However, we had to do this multiple times while debugging the deploy. Whether you want this in the README or not I'm not sure. 

Let me know if further clarification is needed.

Reply:
Oh and it's this secret in the letsencrypt.sh script:
```
apiVersion: v1
kind: Secret
metadata:
  name: letsencrypt-config
  namespace: default
type: Opaque
data: ...
```